packages/webapp/package.json

```diff
@@ -2,17 +2,18 @@
   "name": "litefarm-webapp",
   "version": "3.2.5",
   "description": "LiteFarm Web application",
+  "type": "module",
   "scripts": {
-    "dev": "CYPRESS_COVERAGE=TRUE  vite",
+    "dev": "CYPRESS_COVERAGE=TRUE vite --host",
```

Comment:
@JimmyRowland  
Just starting to have time to review this exciting PR! Here's my first question:

The dev script seems to be used on the Github action `cypress_tests.yml`:  is `--host` appropriate for a CI environment?

Reply:
I think it's fine